### PR TITLE
Cleanup Readme.

### PR DESCRIPTION
**Pull request by aswinsuryan**
Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,26 +2,10 @@
...
-## To run end-to-end tests:
-```bash
-   make e2e
-```
-
-## To run unit tests:
-```bash
-   make test
-```
+The lighthouse architecture is explained in detail at [Lighthouse Architecture](https://submariner-io.github.io/architecture/components/lighthouse/).
```
Comment (tpantelis) [inline README.md:5]: This link has changed to [Service Discovery](https://submariner-io.github.io/architecture/service-discovery/)

Comment (aswinsuryan) [reply README.md]: Done.

Comment (mangelajo) [reply README.md]: Please @aswinsuryan @tpantelis use relative url links

Comment (mangelajo) [reply README.md]: Lol ignoreme this is lighthouse,

**Pull request merged**